packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -0,0 +1,191 @@
+  ${({ script }) => script && getPica(script)};
+  text-transform: uppercase;
+  color: ${({ headerLabelColor }) => headerLabelColor};
+`;
+
+const TitleWrapper = styled.div`
+  color: ${({ titleColor }) => titleColor};
+  padding: ${GEL_SPACING} 0;
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getPica(script)};
+`;
+
+const SummaryWrapper = styled.p`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getLongPrimer(script)};
+  color: ${C_METAL};
+  padding-bottom: ${GEL_SPACING_DBL};
+  margin: 0; /* Reset */
+`;
+
+const ButtonWrapper = styled.div`
+  ${({ service }) => service && getSansRegular(service)};
+  ${({ script }) => script && getMinion(script)};
+  padding: ${GEL_SPACING};
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  color: ${({ durationColor }) => durationColor};
+`;
+
+const IconWrapper = styled.span`
+  > svg {
+    color: ${({ durationColor }) => durationColor};
+    fill: currentColor;
+    width: 1.0625rem;
+    height: ${GEL_SPACING_DBL};
+    margin: 0;
+  }
+`;
+
+const DurationWrapper = styled.span`
+  ${({ dir }) =>
+    dir === 'ltr'
+      ? `padding-left: ${GEL_SPACING};`
+      : `padding-right: ${GEL_SPACING};`}
+`;
+
+const programStateConfig = {
+  live: {
+    backgroundColor: C_POSTBOX,
+    headerLabelColor: C_POSTBOX,
+    headerTextColor: C_EBON,
+    titleColor: C_SHADOW,
+    durationColor: C_WHITE,
+  },
+  next: {
+    backgroundColor: C_WHITE,
+    headerLabelColor: C_KINGFISHER,
+    headerTextColor: C_METAL,
+    titleColor: C_METAL,
+    durationColor: C_KINGFISHER,
+  },
+  onDemand: {
+    backgroundColor: C_EBON,
+    headerTextColor: C_EBON,
+    titleColor: C_SHADOW,
+    durationColor: C_WHITE,
+  },
+};
+
+const ProgramCard = ({
+  dir,
+  service,
+  script,
+  brandTitle,
+  summary,
+  episodeTitle,
+  duration,
+  state: { type, translation },
+  link,
+}) => (
+  <CardWrapper>
+    <TextWrapper dir={dir}>
+      <Link href={link}>
+        <HeadingWrapper
+          service={service}
+          script={script}
+          {...programStateConfig[type]}
+        >
+          {type !== 'onDemand' && (
+            <LabelWrapper
+              service={service}
+              script={script}
+              {...programStateConfig[type]}
+            >
+              {`${translation} `}
+            </LabelWrapper>
+          )}
+          {brandTitle}
+        </HeadingWrapper>
+      </Link>
+      <TitleWrapper
+        service={service}
+        script={script}
+        {...programStateConfig[type]}
+      >
+        {episodeTitle}
+      </TitleWrapper>
+      <SummaryWrapper service={service} script={script}>
+        {summary}
+      </SummaryWrapper>
+    </TextWrapper>
+    <ButtonWrapper
+      dir={dir}
+      service={service}
+      script={script}
+      {...programStateConfig[type]}
+    >
+      <IconWrapper dir={dir} {...programStateConfig[type]}>
```

Comment:
why is the `IconWrapper` and `ButtonWrapper` not making use of the `dir` prop, even though it's being passed in to both ??

Reply:
Nice catch.
Initially, we were applying paddings conditionally based on the `dir`. We no longer need to pass this prop to those components

Reply:
Thanks @rhenshaw56 . This is has been fixed.